### PR TITLE
2 navbar section

### LINKED ISSUE #2: [NEW] Navbar
image

1. Make it responsive (Mobile first)
2. Do not add gradients

### PR DESCRIPTION
**Pull request by pranaytiwariii**
[ADDED]NAVBAR

fixed #2 

- added navbar component folder in src folder.
- added responsiveness to the navbar.
- mapped all the data.

laptop view
image

mobile view
image

image

### CODE REVIEW

File: public/Navbar/close.svg
Comment (kamalika0363) [inline public/Navbar/close.svg:1]: changes color of this cross icon into some lighter shade

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      link:"/"
+    },
+    {
+      id: 4,
+      header: "Team",
+      link:"/"
+    },
+  ];
+  return (
+    <div className="px-6 pt-4 pb-2 flex justify-between text-white">
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:40]: className="px-6 pt-4 pb-2 flex justify-between text-[#AFAFAF]"

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      <div className="flex items-center">
+        <img src={logo} className="h-8 w-8" alt="" />
+        <p className="text-sm ">TCET OpenSource</p>
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:43]: className="text-lg font-medium text-white"

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      </div>
+      {/* ----------full screen---------- */}
+      <div className="flex">
+        <ul className="hidden lg:flex lg:gap-6 text-sm items-center">
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:47]: className="hidden lg:flex lg:gap-6 text-md items-center"

File: src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+          {navLink.map((step) => (
+            <div key={step}>
+              <figure>
+                <li>
+                  <a className="hover:underline-offset-8" href={step.link}>
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:52]: className="hover:underline-offset-8 hover:text-white"

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                    {step.header}
+                  </a>
+                </li>
+              </figure>
+            </div>
+          ))}
+        </ul>
+      </div>
+      <div className="hidden lg:flex lg:gap-2">
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:61]: <div className="hidden lg:flex lg:gap-2  font-medium text-white">

File: src/components/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        <button className="rounded-xl px-4 py-2 border-2 border-gray-500 text-lg">
+          <div className="flex gap-1.5">
+            <img src={discord} alt="" />
+            <a href="">Discord</a>
+          </div>
+        </button>
+        <button className="text-lg  px-4 py-2 rounded-lg mt-0 bg-indigo-500">
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:68]: className="text-lg px-4 py-2 rounded-lg mt-0 bg-[#4C58FE] "

File: src/components/Navbar/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        </button>
+      </div>
+      {/* ----------tablet---------- */}
+      <div className="z-30 lg:hidden items-center" onClick={handleClick}>
+        <img className="h-8 w-8" src={toggle ? close : menu} alt="" />
+      </div>
+      <ul
+        className={
+          toggle
+            ? "navLinks absolute flex flex-col gap-4 text-lg w-full px-8 mt-8 lg:hidden"
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:79]: ? "navLinks absolute flex flex-col gap-4 text-md w-full px-8 mt-12 lg:hidden"

File: src/components/Navbar/Navbar.jsx (hunk 5 above)
Comment (pranaytiwariii) [reply src/components/Navbar/Navbar.jsx:61]: kya changes chiye idhar?

File: src/components/Navbar/Navbar.jsx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+      link:"/"
+    },
+    {
+      id: 4,
+      header: "Team",
+      link:"/"
+    },
+  ];
+  return (
+    <div className="px-6 pt-4 pb-2 flex justify-between text-[#AFAFAF]">
```
Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:40]: className="px-16 py-4 flex justify-between text-[#AFAFAF]"

Comment (kamalika0363) [reply src/components/Navbar/Navbar.jsx:40]: @pranaytiwariii

Comment (kamalika0363) [inline src/components/Navbar/Navbar.jsx:40]: className="p-6 md:px-16 md:py-4 flex justify-between text-[#AFAFAF]"

**Pull request merged**